+/accounts.html

```diff
@@ -0,0 +1,232 @@
+        cursor: pointer;
+    }
+    .pill .reference {
+        top: -25%;
+        margin-left: 8px;
+    }
+    .pill .reference, .pill .forget {
+        display: inline-block;
+    }
+    .pill .forget {
+        width: 24px;
+        height: 24px;
+        border: 1px solid #999;
+        border-radius: 50%;
+        cursor: pointer;
+    }
+    .pill .forget:hover {
+        border: 1px solid red;
+    }
+
+    .pill .forget:hover:before,
+    .pill .forget:hover:after {
+        background: red;
+    }
+
+    .pill .forget::before,
+    .pill .forget::after {
+        content: "";
+        background: #999;
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        width: 12px;
+        height: 2px;
+        transform: translate(-50%, -50%) rotate(45deg);
+    }
+    .pill .forget::after {
+        transform: translate(-50%, -50%) rotate(-45deg);
+    }
+
+    .pill + .pill {
+        margin-inline-start: 8px;
+    }
+
+    /* modifying parent stylesheet */
+    .remember, .submit {
+        background: #eee;
+    }
+    .send {
+        padding: 0;
+    }
+    .send .submit button, .send .submit input {
+        color: initial;
+        background: #eee;
+    }
+
+</style>
+<div id="pillbox">
+
+</div>
+<div class="pill">
+    <div class="reference"><!-- fill in with account's nickname via jquery --></div>
+    <div class="forget" title="forget this account"></div>
+</div>
+<script>
+/*
+Methods globalized in remember.html include:
+    'details' to get the current contents of the check
+    'nickname' to create a bankname-last4 reference to an account
+    'overwrite' to write account details to the DOM
+
+On load, we're going to grab the Proprietor blob from localstorage.
+Makes a chioce here to only pass data via encrypted localstrorage,
+so having to go through enc/dec with any state modification.
+
+For each key in Proprietor.accounts, create a pill by cloning and filling in details, and binding the interaction...
+*/
+;(async function(){
+    // insert the pillbox even if its not being used... might want to style it differently when its empty
+    $('#pillbox').insertAfter('.send .remember');
+
+    app.accounts = {};
+    app.current = null;
+
+    app.addAccount = async function () {
+        var account = app.details();
+        if(!account){
+            // then details error'd, nothing else to do here, box will be unchecked by err()
+            return;
+        }
+        // create a reference
+        var reference = app.nickname(account);
+        app.accounts[reference] = account;
+        app.current = reference;
+
+        memoizeAccounts();
+        renderAccounts();
+    }
+
+    /* remember.html also runs this conditional onload in order to overwrite the check from the default account */
+    if(localStorage.Proprietor && $(".flip").hasClass("flipit") === false){
+        var {accounts, current} = await SEA.decrypt(localStorage.Proprietor, 'rest');
```

Comment:
code looking great so far, lovely actually! Nice reuse of function calls.

Tho... sorry to be annoying here, try to avoid ES6 destructuring `{foo, bar} =`. My style preference is I often have a `var tmp` at the top of a module or function, and then I reuse/reassign it a bunch for temporary things. You don't have to do this, feel free to experiment with other approaches and I'll either 👍 or 👎 . Thanks!!

Reply:
fixed